package.json

```diff
@@ -59,7 +59,7 @@
     "react-split-pane": "0.1.89",
     "react-table": "6.10.3",
     "react-tabs": "3.0.0",
-    "reactstrap": "7.1.0",
+    "reactstrap": "8.2.0",
```

Comment:
We need to hold off on upgrading reactstrap until this bug is resolved: https://github.com/reactstrap/reactstrap/issues/1472